plugins/source-terasology-modules/gatsby-node.js

```diff
@@ -0,0 +1,169 @@
+const { graphql } = require("@octokit/graphql");
+const { DateTime } = require("luxon");
+
+const PLUGIN_NAME = "source-terasology-modules";
+
+const query = `
+query Modules($cursor:String) {
+  organization(login: "Terasology") {
+    repositories(first: 20, after: $cursor) {
```

Comment:
do we want this to be configurable?

Reply:
Hm, not sure about that. It is kind of an optimization one could do to weight off request count vs payload size per request. Not sure what the sweet spot is, but I think it is in any case bound to the query we are sending, which is hard-coded in the plugin. 

Thus, I think keeping it internal and not offering a setting for it is okay.